assets/src/data/eventespresso/lists/test/entities.js

```diff
@@ -87,12 +87,12 @@ describe( 'createEntityResolvers()', () => {
 		[
 			'getEvents',
 			'testQueryA',
-			{ path: '/wp-json/ee/v4.8.36/events?testQueryA' },
+			{ path: '/ee/v4.8.36/events?testQueryA' },
 		],
 		[
 			'getDatetimes',
 			'testQueryB',
-			{ path: '/wp-json/ee/v4.8.36/datetimes?testQueryB' },
+			{ path: '/ee/v4.8.36/datetimes?testQueryB' },
```

Comment:
So I realized while working on this branch that I was actually bundling our scripts with the `@wordpress/api-fetch` package instead of using the `wp.apiFetch` global exposed by WP. That global is automatically initialized to add the `wp-json` prefix to everything.  So, once I fixed that, I didn't need to add the prefix here.